web/beacon-app/src/features/projects/components/APIKeysTable.tsx

```diff
@@ -61,26 +54,12 @@ export const APIKeysTable = ({ projectID }: APIKeysTableProps) => {
     />;
   }
 
-  const statusIconMap = {
-    [APIKEY_STATUS.ACTIVE]: <ConfirmedIndicatorIcon />,
-    [APIKEY_STATUS.INACTIVE]: <PendingIndicatorIcon />,
-    [APIKEY_STATUS.REVOKED]: <RevokedIndicatorIcon />,
-    [APIKEY_STATUS.UNUSED]: <UnusedIndicatorIcon />,
-  };
-
   const initialColumns: any = [
     { Header: t`Key Name`, accessor: 'name' },
     { Header: t`Permissions`, accessor: 'permissions' },
     {
       Header: t`Status`,
-      accessor: (key: { status: APIKeyStatus }) => {
-        return (
-          <div className="flex items-center">
-            {statusIconMap[key.status]}
-            <span className="ml-1">{capitalize(key.status)}</span>
-          </div>
-        );
-      },
+      accessor: 'status',
```

Comment:
The `Status` now appears in bold and only `Active` has a color when I view the tables. 

https://www.awesomescreenshot.com/image/39966744?key=e0f77f2c40bd9b5869ff14e28697e261
https://www.awesomescreenshot.com/image/39966752?key=1b272edf4715b0773375fbbf2aa34d2f
https://www.awesomescreenshot.com/image/39966757?key=e54f8c20e3d5782ede6b0e91eb99d546
https://www.awesomescreenshot.com/image/39966762?key=4efa47510d9e5b3b89defb2874d23a1e